gnuradio-runtime/lib/thread/thread.cc

```diff
@@ -166,7 +166,7 @@ namespace gr {
 
 
 #elif defined(macintosh) || defined(__APPLE__) || defined(__APPLE_CC__) || \
-    defined(__FreeBSD__)
+  defined(__FreeBSD__) || defined(__FreeBSD_kernel__) || defined(__GNU__)
```

Comment:
not sure why `__GNU__` is in here and why there is an `#else` case below. I think `__GNU__` is intended to go into the `#else` case.

Reply:
AFAIK, that's GNU Hurd: https://sourceforge.net/p/predef/wiki/OperatingSystems/

Reply:
Is fine then.